doc/development/development.md

```diff
@@ -0,0 +1,5 @@
+# Development
```

Comment:
would Contributing be a better name? development could refer to game development in general

Reply:
On that topic; it would be nice to include the `CONTRIBUTING.md` file in this category too.

Reply:
I think development is fine as a topic, since contributing should refer to the specific contributing.md doc.